index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html> 
+<html> 
+    <head> 
+        <meta charset='utf-8'> 
+        <meta http-equiv='X-UA-Compatible' content='IE=edge'> 
+        <title>Survey Form Microverse</title> 
+        <meta name='viewport' content='width=device-width, initial-scale=1'> 
+        <link rel='stylesheet' type='text/css' href='style.css'>
+    </head> 
+    <body>
+        <div id="container">
+        <h1 id="title">Microverse Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform.</p>
+        <Div id="Formbox">
+        <form id="survey-form">
+            <label for="Name">Name</label>
+            <input id="input" type="Text" placeholder="Enter your name" name="Name">
+          <label for="Email">Email</label>
+            <input id="input" type="email" placeholder="Enter your Email" name="Email" required>
+            <label for="Phone">Age</label>
+            <input type="number" placeholder="Age" name="Age">
+            <label for="role">Which option best describes your current role?</label>
+        <select name="role" id="roles">
+                <option value="Student">Student</option>
+                <option value="Full time job">Full time job</option>
+                <option value="Full time learner">Full time learner</option>
+                <option value="Prefer not to say">Prefer not tosay</option>
+                <option value="Other">Other</option>
```

Comment:
you should elaborate further by explaining why you think ul tag is better to be used, this will help the developer learn and also understand your point better.